src/main/resources/static/views/home/home.html

```diff
@@ -10,46 +10,29 @@
             <li><a href="tel:6135202600" class="entypo-call"><span>Call</span></a></li>
         </ul>
     </aside>
-    <div class="row">
+
+    <div class="row hometext">
+        <img class="carletonlogo" src="../../img/logo-cu-white-outlined.svg" height="50px"/>
         <h1 class="mainHeader">Welcome to the Capstone Portal</h1>
         <p class="homeHeader">Browse the list of projects and manage your 4th year project.</p>
-        <p class="homeSubHeader">Click the button below to get started</p>
-    </div>
-    <div class="row">
-        <div class="col-md-4 col-md-offset-4">
-            <div class="material-button-anim">
-                <ul class="list-inline" id="options">
-                    <li class="option">
-                        <a href="/projects">
-                            <button class="material-button option1" type="button" title="Projects">
-                                <span class="fa fa-rocket" aria-hidden="true"></span>
-                            </button>
-                        </a>
-                    </li>
-                    <li class="option">
-                        <a href="/login">
-                            <button class="material-button option2" type="button" title="Login | Register">
-                                <span class="fa fa-sign-in" aria-hidden="true"></span>
-                            </button>
-                        </a>
-                    </li>
-                    <li class="option">
-                        <a href="#">
-                            <button class="material-button option3" type="button" title="Announcements">
-                                <span class="fa fa-newspaper-o" aria-hidden="true"></span>
-                            </button>
-                        </a>
-                    </li>
-                </ul>
-                <button class="material-button material-button-toggle" type="button" title="Click Me!">
-                    <span class="fa fa-plus-square" aria-hidden="true"></span>
-                </button>
+        <div class="panel-body" style="text-align: center;">
+            <div class="row" style="display: inline-block;">
+                <div class="col-sm-4">
+                    <p><a href="/projects" class="homeicon"><i style="font-size: 25px;" class="icon-rocket" aria-hidden="true"></i></a><br>Projects</p>
+                </div>
+                <div class="col-sm-4">

```

Comment:
can we put an ng-click on the div to navigate to the routes. Currently, it only works if i click the icon 👍

Reply:
That is intended